### PR TITLE
ccextractor 0.94

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by alebcay** [comment]
I have updated the broken patch, can try using patch from https://raw.githubusercontent.com/alebcay/formula-patches/e5fddd607fb4e2b6b16044eb47fa3407d4d1fdb0/ccextractor/unbundle-libs.patch which should be updated for 0.94 if you want to verify CI build (I have tested locally). I have also opened https://github.com/Homebrew/formula-patches/pull/412 to update the patch we are currently carrying.

**Comment by chenrui333** [comment]
let me try your patch locally.

**Comment by chenrui333** [comment]
@alebcay your patch works for me. and I have approved your PR.

**Comment by alebcay** [comment]
Well, I did not test Linux yet 😁 I will be updating my branch/the formula patch PR soon after I test that.

**Comment by chenrui333** [comment]
Thanks!

**Comment by alebcay** [comment]
@chenrui333 with some adjustments to the formula instead, looks like the new patch is working as-is.

File: Formula/ccextractor.rb [hunk 1]
```diff
@@ -25,15 +25,15 @@ class Ccextractor < Formula
...
+  resource "homebrew-test.mxf" do
     url "https://raw.githubusercontent.com/alebcay/example-artifacts/5e8d84effab76c4653972ef72513fcee1d00d3c3/mxf/test.mxf"
     sha256 "e027aca08a2cce64a9fb6623a85306b5481a2f1c3f97a06fd5d3d1b45192b12a"
   end
 
   # Patch build script to allow building with Homebrew libs rather than upstream's bundled libs
   patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/9bc4ef5a88b9a4d55dead30130aa79f8eee5faf7/ccextractor/unbundle-libs.patch"
-    sha256 "b610950e4ae54a8fce3f5952be6d909cb9790a9c46ff356f83e8d8255c7f1ed1"
+    url "https://raw.githubusercontent.com/alebcay/formula-patches/e5fddd607fb4e2b6b16044eb47fa3407d4d1fdb0/ccextractor/unbundle-libs.patch"
```
Comment (Bo98) [inline Formula/ccextractor.rb:35]:
```suggestion
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/e5fddd607fb4e2b6b16044eb47fa3407d4d1fdb0/ccextractor/unbundle-libs.patch"
```

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**